web-common/src/components/floating-element/WithTogglableFloatingElement.svelte

```diff
@@ -11,8 +11,8 @@
   export let pad = 8;
   export let suppress = false;
   export let active = false;
-
   export let inline = false;
+  export let overflowFlipY = true;
```

Comment:
Do we want this to be `true` here? 

https://github.com/rilldata/rill/assets/4402679/778805b0-2559-41cb-a30b-9669824f61ce

Reply:
Using more instances of `overflowFlipY = false` is definitely something to consider. Given we use `WithTogglableFloatingElement.svelte` all over the app, I didn't want to change its behavior wholesale and have any unintended consequences. I'd rather keep this PR scoped to the specific customer complaint.